src/cookbooks/looker/performance_caching.md

```diff
@@ -0,0 +1,75 @@
+# Optimizing Looker Performance - Caching
+
+This how-to guide is about improving performance of Looker dashboards and explores using caching. This is particularly
+useful if you have many users accessing a dashboard or explore, especially across timezones.
+
+This guide has two alternatives: Applying Generated Datagroups and Custom Datagroups
+
+> ⚠️ Some steps in this guide require knowledge of LookML and a developer license - ask in #data-help for assistance if
+> needed.
+
+The default setting in Looker is to store (cache) the results of any query for one hour. The majority of our data
+is updated at least daily. This means that while users will get the same result for a query, they might have to wait
+for the query to retrieve results from our BigQuery warehouse.
+
+We solve this by using Looker's [datagroups](https://cloud.google.com/looker/docs/caching-and-datagroups). In short,
+datagroups are a method for scheduling various actions in Looker like resetting the cache, e-mailing dashboards, or
+rebuilding derived tables.
+
+## Alternative One - Applying Generated Datagroups
+
+For a number of tables in our warehouse, we automatically generate datagroups in looker-hub.
```

Comment:
I wonder if there is a reason to generate them for some and not for all tables, e.g. using the metadata to set the cache period based on the DAG update.

Reply:
These are generated based on tables referenced in lookml-generator. There are some edge cases where they aren't generated but they should be generated for most straightforward lookml views